tools/bin/acceptance_test_kube_helm.sh

```diff
@@ -65,10 +66,12 @@ if [ -n "$CI" ]; then
 #  trap "mkdir -p /tmp/kubernetes_logs && write_all_logs" EXIT
 fi
 
-kubectl port-forward svc/airbyte-server-svc 8001:8001 &
+kubectl expose $(kubectl get po -l app.kubernetes.io/name=server -o name) --name exposed-server-svc --type NodePort --overrides '{ "apiVersion": "v1","spec":{"ports": [{"port":8001,"protocol":"TCP","targetPort":8001,"nodePort":8001}]}}'
+# kubectl port-forward svc/airbyte-server-svc 8001:8001 &
+# ./tools/bin/health_check.sh &
 
 echo "Running worker integration tests..."
-SUB_BUILD=PLATFORM  ./gradlew :airbyte-workers:integrationTest --scan
+KUBE=true SUB_BUILD=PLATFORM LOG_LEVEL=DEBUG  ./gradlew :airbyte-workers:integrationTest --scan
```

Comment:
I genuinely don't know.  is Debug too noisy for our users or is there useful feedback there?

Reply:
yeah i think we should remove `LOG_LEVEL=DEBUG`
Can always add it back for debugging a specific job
but it would add a lot of noise by default